src/corner-kick/README.md

```diff
@@ -0,0 +1,57 @@
+# Corner Kick
+
+Corner Kick is an Electron app that allows for the visualization of UBC Thunderbots' simulator.
+
+## Contribute
+
+### Node
+
+You will need Node to contribute to this project. To install Node, run the following:
+
+```
+sudo apt-get install nodejs npm
+```
+
+### Yarn
+
+Yarn is considered a more powerful package manager than npm. You should install it by running:
+
+```
+npm install -g yarn
+```
+
+### Installing the project
+
+With Node installed, clone the project and install the dependencies.
+
+```
+yarn install
+```
```

Comment:
Hmm, ideally we don't have to add a new dependency manager. Right now we've managed to get it to the point where a new user has to just run `setup_software.sh` and then they (_should_) be good to go, which is great for new members. Any thoughts on how we could integrate all the dependency management stuff with our current setup?

Reply:
Pushed a new change. _Hopefully_, it should now gather node dependencies and build a release version when running `catkin_make`. For development, you would still have to run `yarn install` and `yarn start`, but I'm open to a better process.